⚠️ DON'T MERGE BEFORE MAINNET - Adjusting READMEs for mainnet release (#2489)
raiden-network/light-client: README.md

  <br/>
  <a href='https://raiden.network/'><img 
      width='400px' 
      alt='Raiden Logo' 
      src="https://user-images.githubusercontent.com/35398162/54018436-ee3f6300-4188-11e9-9b4e-0666c44cda53.png" /></a>
  <br/>
  Raiden Light Client SDK, CLI and dApp
  <br/>
</h2>

<h4 align="center">

>   JavaScript SDK, CLI and dApp to carry out fast, cheap, scalable off-chain token transfers with other <a href="https://github.com/raiden-network/raiden">Raiden Clients on Ethereum testnets and mainnet.</a>.

Reviewer: ```suggestion
  JavaScript SDK, CLI and dApp to carry out fast, cheap, scalable off-chain token transfers with other <a href="https://github.com/raiden-network/raiden">Raiden Clients</a>.
  
 I would delete it here
```

The Raiden Light Client SDK is a [Raiden Network](https://raiden.network) compatible client written in JavaScript/Typescript, capable of running in modern web3-enabled browsers, wallets and Node.js environments.

The Raiden CLI is a reference implementation that provides a HTTP REST server which is fully compatible with the [Raiden API specification](https://docs.raiden.network/raiden-api-1/resources).

The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Light Client SDK, which can be used with web3 wallets like [Metamask](https://metamask.io/) (Desktop) or [imToken](https://token.im/download) (mobile).

> > **INFO:** The Light Client SDK, CLI and dApp are all **work in progress** projects. All three projects have been released for mainnet and all code is available in the [Light Client repository](https://github.com/raiden-network/light-client).

Reviewer: ```suggestion
> **INFO:** The Light Client SDK, CLI and dApp are all **work in progress** projects. All three projects have been released for mainnet and all code is available in the [Light Client repository](https://github.com/raiden-network/light-client).  As this release still has its limitations and is a beta release, it is crucial to read this readme including the security notes carefully before using the software.
```

These step-by-step instructions will guide you through the process for trying out the Raiden demo dApp.

The dApp is hosted on [https://lightclient.raiden.network](https://lightclient.raiden.network/) and in this quick start guide we will be making a transfer using the Light Client on both mainnet and the Goerli testnet with the [MetaMask](https://metamask.io) wallet.

Fot a more in depth user guide, see the [Light Client documentation](https://lightclient.raiden.network/docs/).

### Prerequisites

- MetaMask needs to be installed in your browser and can be [downloaded from their website](https://metamask.io/).
- An ETH balance. for mainnet you'll have to purchase ETH, for the Goerli testnet you can acquire some test ETH by:  

  1. By visiting a Goerli faucet either at [https://faucet.goerli.mudit.blog](https://faucet.goerli.mudit.blog) or [https://goerli-faucet.slock.it/](https://goerli-faucet.slock.it/).
  2. Follow the instructions on respective faucet on how to acquire the ETH.

### Making a First Transfer

1. **Connect to the dApp**  

>    - Mainnet  

Reviewer: A good approach to have the two guides here :) I would make two separate sections as an outer braket. First for mainnet and second for testnet.

Author: Ok, I'll split it into two guides.
